src/toasts/IncomingCallToast.tsx

```diff
@@ -133,13 +133,16 @@ export function IncomingCallToast({ notificationEvent }: Props): JSX.Element {
         setConnectedCalls(Array.from(CallStore.instance.connectedCalls));
     });
     const otherCallIsOngoing = connectedCalls.find((call) => call.roomId !== roomId);
-    // Start ringing if not already.
+    const soundHasStarted = useRef<boolean>(false);
     useEffect(() => {
+        // Start ringing if not already.
+        // This section can race, so we use a ref to keep track of whether we have started trying to play.
```

Comment:
This is a very tight race in my experience (usually a few ms) in it, so even using a `setState` wouldn't stop it from rerendering, hence the ref.

Reply:
What does this effect race with? Would appreciate that being added to the comment.